Jenkinsfile

```diff
@@ -16,14 +16,13 @@ def pip_install_args = "--index-url ${pip_index} --progress-bar=off"
 bc0 = new BuildConfig()
 bc0.nodetype = 'linux'
 bc0.name = 'wheel-sdist'
-bc0.conda_ver = '4.6.8'
+bc0.conda_ver = '4.6.14'
 bc0.conda_packages = [
     "python=${python_version}",
-    "pip<19.1"
 ]
 bc0.build_cmds = [
-    "pip install ${pip_install_args} numpy",
-    "pip wheel ${pip_install_args} .",
+    "pip install numpy",
+    "pip wheel .",
```

Comment:
Just for my own education, what is "pip wheel"?

Reply:
A `wheel` is the modern binary python package distribution format. It replaced `egg` a while back.  `.whl`.  This makes sure that our package and its dependencies can be built into a wheel.  I.e. it makes sure that we can distribute the package to others.

Reply:
In your own checkout of the repo you can
```
python setup.py sdist bdist_wheel
```
and it will build the `sdist` (source distribution) and `bdist_wheel` (binary wheel distribution) in the `build` directory.  These are the things that get uploaded to pypi.org in oder to do a proper release when the day comes.

The command
```
pip wheel .
```
builds not only the wheel for `jwst`, but also the wheels for all the dependencies.

Did I get all that right @jhunkeler?

Reply:
Hmmm, so if it used to be called "egg", why didn't they name the new one "chicken", instead of "wheel"?

Reply:
🐍 🥚